policyengine_us/parameters/gov/states/ct/tax/income/additions/additions.yaml

```diff
@@ -0,0 +1,19 @@
+description: Connecticut counts these sources as additions.
+values:
+  2022-01-01:
+    - us_govt_interest
```

Comment:
create a variable similar to `in_section_179_expense_add_back`

`ct_section_179_expense_add_back` - add the 80% limitation to the documentation but not the actual calculation 
add comment - " # use federal variables if they are added later "

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/890e83f2-feba-4381-96c1-8640b427024c)